web/src/HelpDialog.test.tsx

```diff
@@ -1,12 +0,0 @@
-import { mount } from "enzyme"
-import React from "react"
-import { MemoryRouter } from "react-router-dom"
-import { DialogOverview } from "./HelpDialog.stories"
-
-it("renders overview dialog", () => {
-  mount(
-    <MemoryRouter initialEntries={["/"]}>
-      <DialogOverview />
-    </MemoryRouter>
-  )
-})
```

Comment:
this test didn't have any assertions, so i deleted it. (plus the component itself is mostly just rendering data with the shared dialog component, so there's not really functionality to test.)